lib/Gedmo/Tree/Strategy/ORM/MaterializedPath.php

```diff
@@ -23,12 +23,18 @@ public function removeNode($om, $meta, $config, $node)
         $wrapped = AbstractWrapper::wrap($node, $om);
 
         $path = addcslashes($wrapped->getPropertyValue($config['path']), '%');
+        $lvl = $wrapped->getPropertyValue($config['level']);
```

Comment:
actually there might be an issue if level is not configured. you should first check it with `isset($config['level'])` like https://github.com/freshcells/DoctrineExtensions/blob/dc0c702cc7b3b7376f9761099f209d8c864c1a1f/lib/Gedmo/Tree/Strategy/AbstractMaterializedPath.php#L316 because it is optional. should be also fixed in master, haven't noticed it before